### PR TITLE
Update workflow-cps, antisamy-markup-formatter, and the PCT

### PR DESCRIPTION
**Pull request by dwnusbaum**
Originally filed to test https://github.com/jenkinsci/workflow-cps-plugin/pull/612 and https://github.com/jenkinsci/plugin-compat-tester/pull/383, but now it updates everything related to those PRs.

### CODE REVIEW

**Comment by dwnusbaum** [comment]
Looks good in general. The only issue so far is with `antisamy-markup-formatter` because https://github.com/jenkinsci/antisamy-markup-formatter-plugin/pull/65 has not yet been released and so it does not support incrementals:

```
[2022-11-03T19:50:29.619Z] [ERROR] Failed to execute goal org.jenkins-ci.tools:maven-hpi-plugin:3.33:resolve-test-dependencies (default-resolve-test-dependencies) on project antisamy-markup-formatter: Unable to resolve dependencies: Could not resolve dependencies for project org.jenkins-ci.plugins:antisamy-markup-formatter:hpi:2.7: Could not find artifact org.jenkins-ci.plugins.workflow:workflow-cps:jar:3513.v1b_01482f7077 in repo.jenkins-ci.org (https://repo.jenkins-ci.org/public/) -> [Help 1]
```

`workflow-cps` itself has passed against the latest weekly, but hasn't run against 2.332.x yet as far as I can tell.

**Comment by dwnusbaum** [comment]
I will push a commit to set `failFast` to `false` so we know for sure whether `antisamy-markup-formatter` is the only problem.

**Comment by jglick** [comment]
> has not yet been released

Hmm, I will fix that.

**Comment by jglick** [comment]
https://github.com/jenkinsci/antisamy-markup-formatter-plugin/releases/tag/155.v795fb_8702324

**Comment by dwnusbaum** [comment]
The GitHub check mentioning that `caffeine-api` failed is just a reporting bug. The only plugin that failed is indeed `antisamy-markup-formatter` in the weekly branch. I will update it and remove `failFast = false` and then I think all of the downstream stuff is ready to be merged.

File: prep.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ for LINE in $LINEZ; do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=1190.v519c09131444
+version=1198.v72d48d41403a # https://github.com/jenkinsci/plugin-compat-tester/pull/383
```
Comment (dwnusbaum) [inline prep.sh:44]:
```suggestion
version=1199.vb2c7632aa27b
```

File: bom-weekly/pom.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <scm-api-plugin.version>621.vda_a_b_055e58f7</scm-api-plugin.version>
         <subversion-plugin.version>2.16.0</subversion-plugin.version>
         <workflow-api-plugin.version>1200.v8005c684b_a_c6</workflow-api-plugin.version>
-        <workflow-cps-plugin.version>2803.v1a_f77ffcc773</workflow-cps-plugin.version>
+        <workflow-cps-plugin.version>3513.v1b_01482f7077</workflow-cps-plugin.version> <!-- TODO: https://github.com/jenkinsci/workflow-cps-plugin/pull/612 -->
```
Comment (dwnusbaum) [inline bom-weekly/pom.xml:24]:
```suggestion
        <workflow-cps-plugin.version>3520.va_8fc49e2f96f</workflow-cps-plugin.version>
```

**Review by jglick — APPROVED** [review]

**Pull request merged**